site/cwlsite-job.yaml

```diff
@@ -1,15 +1,10 @@
 #!/usr/bin/env cwl-runner
 cwl:tool: cwlsite.cwl
-brandimg: {class: File, path: CWL-Logo-Header.png}
+jekyll-site: {class: Directory, location: ../content}
+user_guide: {class: Directory, location: ../user_guide}
+novice-tutorial: {class: Directory, location: ../cwl-novice-tutorial}
+rnaseq-training: {class: Directory, location: ../rnaseq-cwl-training}
```

Comment:
FYI: adding the rnaseq training to www.commonwl.org should not have been part of this PR as I did not notice it, nor was that approved by any process.